src/widget/wpushbutton.cpp

```diff
@@ -444,6 +432,14 @@ bool WPushButton::event(QEvent* e) {
             m_bPressed = false;
             restyleAndRepaint();
         }
+    } else if (e->type() == QEvent::ToolTip) {
+        updateTooltip();
+    } else if (e->type() == QEvent::Enter) {
+        m_bHovered = true;
```

Comment:
Is this hack still required? I had the impression that this is not the case during development of the HotcueButton.

Reply:
yes.
`[hover="true"]` allows to change the font color, default Qt `:hover` doesn't.

Reply:
It was introduced here recently:  https://github.com/mixxxdj/mixxx/pull/2471
So I think it is required ..

Reply:
yup, that PR was from me :) Just tested again: with `:hover` the font won't change.